messaging.rst

```diff
@@ -41,14 +35,23 @@ Fields
 +--------------------------+------------+--------------------------------------------------------------------------------+
 |  channel_identifier      | uint256    | Channel identifier inside the TokenNetwork contract                            |
 +--------------------------+------------+--------------------------------------------------------------------------------+
-|  message_hash            | bytes32    | Hash of the message                                                            |
-+--------------------------+------------+--------------------------------------------------------------------------------+
-|  signature               | bytes      | Elliptic Curve 256k1 signature on the above data                               |
+|  signature               | bytes      | Elliptic Curve 256k1 signature described below                                 |
 +--------------------------+------------+--------------------------------------------------------------------------------+
 |  chain_id                | uint256    | Chain identifier as defined in EIP155                                          |
 +--------------------------+------------+--------------------------------------------------------------------------------+
 
 
+Signature of Balance Proofs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The signature field in the offchain balance proof is exactly the same thing as the signature in the onchain balance proof.  The signature must be valid and is defined as:
+
+::
+
+    ecdsa_recoverable(privkey, keccak256(balance_hash || nonce || additional_hash || channel_identifier || token_network_address || chain_id)
+
+In the signature, ``additional_hash`` witnesses the integrity of the whole Raiden message that contains the balance proof.
```

Comment:
Consider adding a reference to the `additional_hash` terminology definition.

Reply:
That's a good idea.  I will do this in this PR.

Reply:
Since I closed this PR, I have to take care of what's discussed here.

Reply:
So I opened https://github.com/raiden-network/spec/pull/161